Kitematic connection issue - parsing JSON

I have Kitematic on OSX and it installed default VM etc. When I try to connect with this:

		DockerClientConfig config = DockerClientConfig
			.createDefaultConfigBuilder()
			.withVersion("1.18")
			.withDockerCertPath("/Users/igor/.docker/machine/machines/dev")
			.withUri("https://192.168.99.100:2376")
			.build();

		DockerClient dockerClient = DockerClientBuilder
			.getInstance(config)
			.build();

		Info info = dockerClient.infoCmd().exec();

it fails, due to error:

```
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Can not deserialize instance of java.lang.String out of START_ARRAY token
 at [Source: org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream@6a10b263; line: 1, column: 525] (through reference chain: com.github.dockerjava.api.model.Info["Labels"])
```

--- a/src/main/java/com/github/dockerjava/api/model/Info.java
+++ b/src/main/java/com/github/dockerjava/api/model/Info.java
@@ -60,7 +60,7 @@ public class Info {
 	private String kernelVersion;
 	
 	@JsonProperty("Labels")
-	private String Labels;
+	private String[] Labels;
 
 	@JsonProperty("MemoryLimit")
 	private boolean memoryLimit;
@@ -140,7 +140,7 @@ public class Info {
 		return kernelVersion;
 	}
 	
-	public String getLabels() {
+	public String[] getLabels() {
 		return Labels;
 	}
 

